tests/session/transaction-integration-003.phpt

```diff
@@ -0,0 +1,58 @@
+--TEST--
+MongoDB\Driver\Session::startTransaction() Transient Error Test
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
+<?php skip_if_not_clean(DATABASE_NAME, COLLECTION_NAME); ?>
+<?php skip_if_not_replica_set(); ?>
+<?php skip_if_server_version('<', '4.0'); ?>
+--FILE--
+<?php
+require_once __DIR__ . "/../utils/basic.inc";
+
+$manager = new MongoDB\Driver\Manager(URI);
+
+/* Create collections as that can't be (automatically) done in a transaction */
+$cmd = new \MongoDB\Driver\Command([
+    'create' => COLLECTION_NAME,
```

Comment:
I believe this should use a majority write concern. Per [Transactions and Locks](https://docs.mongodb.com/manual/core/transactions/#transactions-and-locks):

> When creating or dropping a collection immediately before starting a transaction, if the collection is accessed within the transaction, issue the create or drop operation with write concern "majority" to ensure that the transaction can acquire the required locks.

Would you mind updating that for all transaction tests? Feel free to make a separate ticket if you like (akin to [PHPLIB-377](https://jira.mongodb.org/browse/PHPLIB-377)), but I'm fine without.

Reply:
https://jira.mongodb.org/browse/PHPC-1244